src/transformStateWithClones.js

```diff
@@ -66,6 +66,32 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const listOfClones = [];
+  const Clone = { ...state };
+
+  for (const object of transforms) {
```

Comment:
transforms - plural
transform - singular

```suggestion
  for (const transform of transforms) {
```